src/pages/Inventory.page.js

```diff
@@ -61,5 +71,29 @@ export class InventoryPage extends BaseSwagLabPage {
         }
      }
 
- 
+   async getProducts() {
+    const productElements = await this.inventoryItems.all();
+    return productElements;
+
+   }
+
+   async addRandomProductsToCart() {
+    const allProducts = await this.getProducts();
+    const randomProducts = await  getRandomProducts(allProducts, 2);
```

Comment:
а чому тільки 2? Потрібно рандомну кількість рандомних :)

Reply:
```suggestion
    const randomProducts =  getRandomProducts(allProducts, 2);
```
await is not needed here as your function is not returning promise